console/console-number-format-specifiers-symbol-manual.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Console Number Format Specifiers on Symbols</title>
+<meta name="author" title="Dominic Farolino" href="mailto:domfarolino@gmail.com">
+<meta name="assert" content="Console format specifiers on Symbols">
+<link rel="help" href="https://console.spec.whatwg.org/#formatter">
+</head>
+<body>
+<p>Open the console inside the developer tools. It should contain 15 lines, each of which are:</p>
+<p><code>NaN</code></p>
+
+<script>
+const methods = ["log", "dirxml", "trace", "group", "groupCollapsed"];
+
+for (method of methods) {
+  console[method]("%i", Symbol.for("description"));
+  if (method == "group" || method == "groupCollapsed") console.groupEnd();
```

Comment:
This isn't the prettiest thing ever, but I'm kinda OK with it unless someone wants me to change it.